pandas/tests/arrays/integer/test_function.py

```diff
@@ -64,6 +64,20 @@ def test_ufuncs_binary_int(ufunc):
     tm.assert_extension_array_equal(result, expected)
 
 
+def test_ufunc_binary_output():
+    a = integer_array([1, 2, np.nan])
+    result = np.modf(a)
```

Comment:
could parametrize over the (few?) functions that could go through here?

Reply:
The binary ufuncs that overlap with dunder methods hit a different code path. I'm not sure of others (and I'm also not sure I see the value in parametrizing over them. Given the current structure of the code they should be redundant).